saleor/graphql/plugins/filters.py

```diff
@@ -1,16 +1,25 @@
-from typing import TYPE_CHECKING, List, Optional
+from typing import List, Optional
 
 import graphene
 
-if TYPE_CHECKING:
-    # flake8: noqa
-    from ...plugins.base_plugin import BasePlugin
+from .types import Plugin
 
 
-def filter_plugin_search(
-    plugins: List["BasePlugin"], value: Optional[str]
-) -> List["BasePlugin"]:
-    plugin_fields = ["PLUGIN_NAME", "PLUGIN_DESCRIPTION"]
+def filter_plugin_is_active(plugins: List[Plugin], is_active) -> List[Plugin]:
```

Comment:
Maybe we should add a channel param to this function to be able to filter through active plugins in a specyfic channel.

Reply:
https://github.com/mirumee/saleor/pull/7205#discussion_r617243342